lib/natalie/compiler/pass4.rb

```diff
@@ -674,6 +674,7 @@ def process_sexp(exp, name = nil, type = 'Value ')
         fn, *args = exp
 
         return "Value::integer(#{args.map { |a| process_atom(a) }.join(', ')})" if fn == :'Value::integer'
+        return "Value::floatingpoint(#{args.map { |a| process_atom(a) }.join(', ')})" if fn == :'Value::integer'
```

Comment:
I think this is supposed to check `Value::floatingpoint`? Honestly don't remember what this code is doing... :grimacing: